build.gradle

```diff
@@ -1,6 +1,6 @@
 plugins {
     id "architectury-plugin" version "3.4-SNAPSHOT"
-    id "dev.architectury.loom" version "0.12.0-SNAPSHOT" apply false
+    id "dev.architectury.loom" version "1.0-SNAPSHOT" apply false
```

Comment:
Is Architectury formally v1.0? Nice! :)

Reply:
Fabric loom is 1.0 and they are following the fabric versioning. But yes, nice.